src/core/lib/tsi/ssl_transport_security.c

```diff
@@ -353,8 +349,8 @@ static tsi_result add_subject_alt_names_properties_to_peer(
         result = TSI_INTERNAL_ERROR;
         break;
       }
-      const char *name = inet_ntop(af, subject_alt_name->d.iPAddress->data,
-                                   ntop_buf, INET6_ADDRSTRLEN);
+      const char *name = grpc_inet_ntop(af, subject_alt_name->d.iPAddress->data,
```

Comment:
And actually, "nope!" :-D

Because it means tsi depends on gpr. We'll have to revert it, sadly, and find sometime better.